core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/test-constants.js

```diff
@@ -32,7 +32,8 @@ let testConstants = {
   },
   SignInDetails: {
     inputField: 'input.e2e-test-sign-in-email-input',
-  }
+  },
+  DEFAULT_SPEC_TIMEOUT: 300000
```

Comment:
Why this value exactly?

Reply:
The value is based off the average time a test can take to complete. Mostly the longest I have seen is around 3-4 min rarely. So the safe bet is around 5 min ie 300000 millia sec

Reply:
Okay